tests/utils/test_interfaces.py

```diff
@@ -78,12 +81,12 @@ def main():
     # We want to avoid catastrophic failure at runtime for a single
     # bad plugin, so ensure these are validated in testing, and allow
     # bad plugins to get through at runtime.
-    plg_reg = plugin_registry.PluginRegistry()
+    plg_reg = PluginRegistryValidator()
     try:
         LOG.interactive("Testing all registries...")
         plg_reg.validate_all_registries()
         successful_registries += ["all"]
-    except plugin_registry.PluginRegistryError as resp:
+    except errors.PluginRegistryError as resp:
         failed_registries += ["all"]
         failed_tracebacks += [
             f"\n\n\n{FAILED_INTERFACE_HEADER_PRE}\n\n{traceback.format_exc()}"
```

Comment:
It might be a good idea to replace this section with what's been implemented in #527.

Reply:
I just copy-pasted the updates from #527 into this branch, and ran test_interfaces again. v1_12_2.rst release notes now include the updates from #527 as well.

Reply:
Feel free to resolve unless you have any other comments on this.

Reply:
Resolving.